.circleci/config.yml

```diff
@@ -90,33 +71,13 @@ jobs:
             mv ~/protoc3/bin/* /usr/local/bin/
             mv ~/protoc3/include/* /usr/local/include/
       - run:
-          name: Get proto dependencies.
-          command: |
-            git clone -b input-contract \
-                https://github.com/googleapis/api-common-protos.git \
-                tmp/api-common-protos
-      - run:
-          name: Stage the showcase protos with it's dependencies.
-          command: |
-            mkdir -p tmp/api-common-protos/google/showcase/v1alpha2
-            cp schema/echo.proto tmp/api-common-protos/google/showcase/v1alpha2/
-      - run:
-          name: Compile the proto descriptor set.
-          command: |
-            protoc --proto_path=tmp/api-common-protos --include_imports \
-                --include_source_info \
-                -o dist/gapic-showcase-v1alpha2-${VERSION}.desc \
-                tmp/api-common-protos/google/showcase/v1alpha2/echo.proto
-      - run:
-          name: Make a tarball of the protos.
+          name: Create release assets
           command: |
-            mkdir tmp/gapic-showcase-v1alpha2-${VERSION}-protos
-            cp -r tmp/api-common-protos/google tmp/gapic-showcase-v1alpha2-${VERSION}-protos
-            cd tmp/ && tar -zcf ../dist/gapic-showcase-v1alpha2-${VERSION}-protos.tar.gz gapic-showcase-v1alpha2-${VERSION}-protos
+            go get ./util/release
```

Comment:
I don't think you actually need this line.

Reply:
Done.